client/components/conditions/index.js

```diff
@@ -44,19 +42,11 @@ function ConditionsPage({
       {
         !editConditions && !conditions.length && <ReactMarkdown className="grey">{emptyIntroReadOnly}</ReactMarkdown>
       }
-      {
-        showRA && (
-          <Fragment>
-            <h2>Additional condition 1</h2>
-            <RetrospectiveAssessment />
-          </Fragment>
-        )
-      }
       <Conditions
         scope="project"
         type={type}
         conditions={conditions}
-        startIndex={showRA ? 1 : 0}
```

Comment:
It's highly likely that this is no longer required. If it's not then we should remove the prop completely.